Moved to new versions of cvc4 and z3

pjljvandelaar opened the pull request.

pjljvandelaar committed:
Moved to new versions of cvc4 and z3

pjljvandelaar committed:
script contained flaws -- repaired

pjljvandelaar committed:
new z3 has an additional directory

pjljvandelaar committed:
Z3 has additional directory

Script tries to unzip and move to standard z3 directory.

pjljvandelaar committed:
Another attempt

No linux at home to try offline

pjljvandelaar committed:
Z3 iso ./z3

pjljvandelaar committed:
Target is not a directory

Add prefix to solve it....

pjljvandelaar committed:
Prevent endless recursion

dnadales reviewed (commented):
BTW, you can ssh into one of these Linux machines using semaphore and try these commands. Might speed up the process ;)

pjljvandelaar commented:
Thanks for the suggestion

pjljvandelaar committed:
Last attempt for the day

pjljvandelaar committed:
Once more

pjljvandelaar committed:
Update setup.sh

pjljvandelaar committed:
mv is rename and move

multiple objects must be moved to existing directory: I didn't get the error message I guess ;-)

pjljvandelaar committed:
ls gave files in directory

I want the name of the directory not the files in the directory

pjljvandelaar committed:
removed obsolete code + improved code

pjljvandelaar committed:
With Echo it seemed to work

confused once more ;-(

pjljvandelaar committed:
made z3 executable

pjljvandelaar committed:
invalidate cache needed

pjljvandelaar committed:
Don't understand why z3 is no longer found

pjljvandelaar committed:
caches correct now?

pjljvandelaar committed:
New cvc4 has changed flags

Issue was no longer in Z3.
As already written down: https://github.com/TorXakis/TorXakis/issues/652
cvc4 has changed flags.

Had to be changed in THREE locations.

pjljvandelaar committed:
more info on setting and versions

pjljvandelaar committed:
cache was polluted with z3-4.7.1 directory -> find found 2

pjljvandelaar committed:
link incorrect - Not found message made executable ;-)

pjljvandelaar commented:
@tretmans Everything is ready for after the release.
Please approve asap.

pjljvandelaar committed:
temp script to clear caches

pjljvandelaar committed:
cvc4 was absent in cache - cache empty?

pjljvandelaar committed:
changed stack.yaml to clean caches on appveyor

pjljvandelaar committed:
cvc4 bug returned - removing cache once more
